### PR TITLE
postsubmit job for codeready-toolchain/host-operator

### PR DESCRIPTION
**Pull request by xcoulon**
2 jobs to:
- run the build
- run the tests with coveage + upload to codecov.io

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: ci-operator/jobs/codeready-toolchain/host-operator/codeready-toolchain-host-operator-master-postsubmits.yaml [hunk 1]
```diff
@@ -1,5 +1,79 @@
...
+    decoration_config:
+      skip_cloning: true
+    name: branch-ci-codeready-toolchain-host-operator-master-e2e
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=e2es
```
Comment (dipak-pawar) [inline ci-operator/jobs/codeready-toolchain/host-operator/codeready-toolchain-host-operator-master-postsubmits.yaml]: mispelled?

Comment (xcoulon) [reply ci-operator/jobs/codeready-toolchain/host-operator/codeready-toolchain-host-operator-master-postsubmits.yaml]: yes! Stupid `cmd` key that I did press hard-enough when saving the file :/

Comment (xcoulon) [reply ci-operator/jobs/codeready-toolchain/host-operator/codeready-toolchain-host-operator-master-postsubmits.yaml]: fixed in 74fd39ad1

File: ci-operator/jobs/codeready-toolchain/host-operator/codeready-toolchain-host-operator-master-postsubmits.yaml [hunk 2]
```diff
@@ -1,5 +1,79 @@
...
+    decoration_config:
+      skip_cloning: true
+    name: branch-ci-codeready-toolchain-host-operator-master-e2e
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=e2e
```
Comment (dipak-pawar) [inline ci-operator/jobs/codeready-toolchain/host-operator/codeready-toolchain-host-operator-master-postsubmits.yaml]: I think, we are missing template and secret-dir

```bash
 - --artifact-dir=$(ARTIFACTS)
        - --give-pr-author-access-to-namespace=true
        - --secret-dir=/usr/local/e2e-cluster-profile
        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
        - --target=e2e
        - --template=/usr/local/e2e
```

Comment (xcoulon) [reply ci-operator/jobs/codeready-toolchain/host-operator/codeready-toolchain-host-operator-master-postsubmits.yaml]: ok, let me fix that

Comment (xcoulon) [reply ci-operator/jobs/codeready-toolchain/host-operator/codeready-toolchain-host-operator-master-postsubmits.yaml]: fixed in https://github.com/openshift/release/pull/4421/commits/ef4d0d97cad30759006e9124b9cd3fbe18501d0b

**Comment by xcoulon** [comment]
thanks for the review @dipak-pawar 🙌

**Pull request merged**